src/utils/components/CloneVMModal/CloneVMModal.tsx

```diff
@@ -54,7 +54,7 @@ const CloneVMModal: FC<CloneVMModalProps> = ({ headerText, isOpen, onClose, sour
   const [initialCloneRequest, setInitialCloneRequest] = useState<V1beta1VirtualMachineClone>();
 
   const sendCloneRequest = async () => {
-    const vmSameName = await vmExist(cloneName, namespace);
+    const vmSameName = await vmExist(cloneName, namespace, source?.cluster);
```

Comment:
I know this isn't part of this PR, but could you please rename `vmExist` to `vmExists`? It looks like it's only used in one or two places, so it won't hurt anything to do it in this PR instead of a followup.